src/list_vault_jwt_expiration/list_vault_jwt_expiration/list.py

```diff
@@ -0,0 +1,102 @@
+import logging
```

Comment:
It's not clear to me why this code is in a sub-package which has the same name as the parent package... is there a reason for that? It seems unnecessarily nested.

Reply:
Happy to refactor this, but would be helpful if I could be pointed to our preferred pattern?

Reply:
I think a structure like this is more clear:
```
src/
├─ list_vault_jwt_expiration/
│  ├─ main.py
│  ├─ list.py
│  ├─ context.py
```
where `main.py` is what the higher-up `list.py` is. `main.py` is common convention for entry-point files so I think it makes the purpose of that file easier to understand. If you really wanted the other files nested in another folder (I personally think you don't need it as there's so few but up to you) you can have `utils/`

Reply:
I agree the above would be more intuitive. The folder is treated as a package and each file in a folder is already a module. You probably don't need sub-packages for a simple script such as this. Packages usually do not look like function names but more like namespaces, so `list_vault_jwt_expiration` is also unconventional.

Reply:
Paired with @amonkhouse yesterday and flattened out the folder structure a bit (see [a8e7198](https://github.com/artsy/opstools/pull/125/commits/a8e71983b4500f300f95f8c363d4c71a6cc33321)). Also discussed that it probably makes sense to take another look at the convection of the project as a whole, but it is outside the scope of this PR, which I have let live too long, as it is 😬.